lib/oxidized/model/edgecos.rb

```diff
@@ -11,7 +11,11 @@ class EdgeCOS < Oxidized::Model
     cfg.each_line.to_a[0..-2].join
   end
 
-  cmd 'show running-config'
+  cmd 'show running-config' do |cfg|
+    cfg.gsub! /^(show running-config)$/, '! \\1'
```

Comment:
Is the intention here to remove the command itself from the input? Perhaps cfg.cut_head would be a more obvious approach?

Reply:
No, I am just avoiding accidental copy/pasting of `show running-config` command to a device.
Commands were recently added to the output in #2024. I have no idea why do we want them in the output, but it is ok for me unless they are not commented out.

Reply:
This is a good point, @freddy36 - could you comment about the utility of including the commands in the output instead of cutting them out? I understand the concern of @moisseev - perhaps we can find a middle ground such as including a comment character before the first line or lines of the output?

Reply:
If we don't receive feedback from @freddy36 here, my preference would be to re-introduce the situation prior to #2024 where the commands were removed from the input, as this has been the implicit default for most models.

Reply:
I agree. Let's wait few more days for their response.

Reply:
We like to have the commands in the output to see which command generated the corresponding output. Commenting out the commands is definitely the best option.
But completely removing them is also fine with me.

Reply:
In that case, I suggest we remove the specific running-config comment/clean-up here, and revert the `:all` section to remove command names from output (as this is not a common behavior for Oxidized models in general).

We should perhaps consider this a feature request for `cmd` to optionally prefix the name of the command executed to the output using the defined comment character of each model as a global and/or device level setting, since I do see the use case @freddy36 mentions as potentially relevant to a broader audience.

Reply:
Done.